marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/jobInfo/jobInfo.api

```diff
@@ -0,0 +1,13 @@
+{
+    "functionName": "latestJobInfo",
```

Comment:
I recommend calling this "getLatestJobData". I think we want data services to have names that are verb-oriented, like method names. I was initially thinking "getLatestJobDataForEntityCollection", but I can see this expanding to support additional parameters.

Reply:
Included in https://github.com/marklogic/marklogic-data-hub/pull/3011/commits/e1d00a2bc5e0cc4d4afe9e00cfd0fc146f4f53aa